scripts/dockerfiles/run.sh

```diff
@@ -90,6 +91,29 @@ done
         /var/vcap/jobs/{{ $job.Name }}/bin/run
     {{ end }}
 {{ else }}
-    # Replace bash with monit to handle both SIGTERM and SIGINT
-    exec dumb-init -- monit -vI
+
+  killer() {
+    # Wait for all monit services to be stopped
+    echo "Received SIGTERM. Will run 'monit stop all'."
+
+    total_services=$(monit summary | grep -c "^Process")
+
+    monit stop all
+
+    echo "Ran 'monit stop all'."
+
+    while [ $total_services != $(monit summary | grep "^Process" | grep -c "Not monitored") ] ; do
```

Comment:
Is there a reason that this isn't `test -z "$(monit summary | grep '^Process' | grep -v 'Not monitored')"`? Just seemed odd to get the count first, that's all.